src/IndexGame.py

```diff
@@ -56,8 +59,15 @@ def displayGameMenu(turn, map, selectedBuilding):
 
 def chooseBuildingPool():
     #To choose which building to build for building pool
+    choices = input("Enter the 5 building types you want: ").split(' ')
+    for choice in choices:
```

Comment:
is it possible to change to accept 1 input at a time? I alr made the UAT and system test scripts and cases that way and it would be a hassle to manually change all of them 1 by 1.

Reply:
Right, sorry about that, forgot about it. Will redo.

Reply:
thanks alot, sorry for the hastle